ci.hocon

```diff
@@ -181,13 +181,39 @@ graal-vm-snapshot: {
   }
 }
 
+jruby-benchmark: {
+  setup: ${common-base.prelude}
+
+  downloads: {
+    JRUBY_HOME: {name: jruby, version: "9.1.12.0"}
+  }
+
+  environment: {
+    HOST_VM: server,
+    HOST_VM_CONFIG: default
+    GUEST_VM: jruby,
+    GUEST_VM_CONFIG: indy,
+    RUBY_BIN: "$JRUBY_HOME/bin/jruby",
+    JT_BENCHMARK_RUBY: "$JRUBY_HOME/bin/jruby",
+    JRUBY_OPTS="-Xcompile.invokedynamic=true"
```

Comment:
What did they tell you about the `all` option?

Reply:
I never heard back from Charlie. Tom said `-Xcompile.invokedynamic=true` is all he ever uses.

Reply:
I did hear back from Charlie and the option we're using is correct.